pkgs/cronet_http/README.md

```diff
@@ -40,22 +38,45 @@ void main() async {
   final Client httpClient;
   if (Platform.isAndroid) {
     final engine = CronetEngine.build(
-        cacheMode: CacheMode.memory,
-        cacheMaxSize: 2 * 1024 * 1024,
-        userAgent: 'Book Agent');
+      cacheMode: CacheMode.memory,
+      cacheMaxSize: 2 * 1024 * 1024,
+       userAgent: 'Book Agent',
```

Comment:
There seems to be some random formatting changes in this file...isn't this formatting actually incorrect?

Reply:
Flutter prefers trailing commas, which makes the formatting different.

Reply:
This just looks misindented though. Can you change it back?

Reply:
Sure